dotCMS/src/main/java/com/dotmarketing/startup/runonce/Task04110AddColumnsPublishingPushedAssetsTable.java

```diff
@@ -0,0 +1,59 @@
+package com.dotmarketing.startup.runonce;
+
+import java.sql.SQLException;
+
+import com.dotmarketing.common.db.DotConnect;
+import com.dotmarketing.db.DbConnectionFactory;
+import com.dotmarketing.exception.DotDataException;
+import com.dotmarketing.exception.DotRuntimeException;
+import com.dotmarketing.startup.StartupTask;
+
+/**
+ * Add the environmentId and publisher columns to the publishing_pushed_assets
+ * @author Oswaldo Gallango
+ *
+ */
+public class Task04110AddColumnsPublishingPushedAssetsTable implements StartupTask {
```

Comment:
We can just extend the AbstractJDBCStartupTask class instead, since it already provides easy methods to execute queries for all DBs.